compiler/rustc_error_codes/src/error_codes/E0783.md

```diff
@@ -0,0 +1,22 @@
+The range pattern `...` is no longer allowed.
+
+Erroneous code example:
+
+```edition2021,compile_fail,E782
```

Comment:
This should be E0783, right?

Reply:
Wow, great catch! It should be indeed! I'm surprised that the linter didn't detect it...